src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -282,9 +290,8 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
 
             gj.setAskedResources(new ResourceSet(null, mem, cores, nodes, walltime, null, it["queue"] as String, additionalNodeFlag))
             gj.setUsedResources(new ResourceSet(null, usedMem, null, null, usedWalltime, null, it["queue"] as String, null))
-
-            gj.setLogFile(getQstatFile(it["Output_Path"] as String))
-            gj.setErrorLogFile(getQstatFile(it["Error_Path"] as String))
+            gj.setLogFile(getQstatFile((it["Output_Path"] as String).replace("\$PBS_JOBID", jobID.toString())))
+            gj.setErrorLogFile(getQstatFile((it["Error_Path"] as String).replace("\$PBS_JOBID", jobID.toString())))
```

Comment:
This is wrong, the placeholder is replaced by the full job id, as it is returned in the field `Job_Id`